packages/oak-addon-basic-components/lib/core/Image/index.styl

```diff
@@ -6,7 +6,6 @@
     align-items: center
 
   .oak-image-preview
-    width: 50px
```

Comment:
this one is here in case flex is not supported correctly (e.g always add width or max-width alongside flex)